### DISCUSSION TITLE
样式污染

### DISCUSSION THREAD

**Discussion by wangqiqi95** [youzan/vant#11163] [Q&A]
### 重现链接

暂无

### Vant 版本

3.5.4

### 描述一下你遇到的问题。

样式污染

**Comment by chenjiahan** [comment]
在 SPA 中，不同的子页面之间本身就有这种现象，我理解不是组件库引起的问题，这个 issue 我移动到讨论区哈

**Reply by wangqiqi95** [reply]
在使用其他的ui框架时，支持使用样式穿透配合<style scope>去修改子组件的样式。目前vant暂时不知道如何通过这两种方法去修改单个页面的组件样式

**Reply by chenjiahan** [reply]
Vant 也支持的，直接修改就好，没什么特殊要求

**Reply by wangqiqi95** [reply]
我用的这个版本的vant，需要把样式写在<style></style>里才能生效。<style scope>这种形式的拿不到子组件，一次都没有修改成功过0.0
